src/data_cell/graph_interface.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright 2024-present the vsag project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "graph_interface.h"
+
+#include "graph_datacell.h"
+#include "io/io_headers.h"
+#include "sparse_graph_datacell.h"
+
+namespace vsag {
+
+std::shared_ptr<GraphInterface>
```

Comment:
add an alias of std::shared_ptr<GraphInterface>, and replace this line with it

Reply:
done